app/views/challenges/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is all he changllenges stands</h1>
```

Comment:
Challenges*

Reply:
i need an other content for this though. Any idea?